site/templates/tv/detail.jet

```diff
@@ -47,20 +47,20 @@
             </div>
           </div>
           <div class="meta-detail-synopsis">{{tvseason.Overview | raw}}</div>
-          <div class="meta-detail-cast">
-          {{if len(tvseason.Cast) > 0 }}
-            <h2>{{i18n("meta_detail_cast_title")}}</h2>
-          {{end}}
-            <p>
-              {{range index, member := tvseason.Cast}}
-                {{ path := "/search.html?q=" + member.Name }}
-                <a href="{{ routeToPath(path) }}">{{member.Name}}</a>{{if index < len(tvseason.Cast) - 1}}, {{end}}
-              {{end}}
-            </p>
-          </div>
-          <div class="meta-detail-crew d-flex">
+          <div class="meta-detail-infos">
```

Comment:
Can you please add the element switcher to this file too and move the cast `if` statement outside the cast div to match the film detail page.

Reply:
- element switcher functionality added to tv detail
- if statement matched to film detail (outside cast)